go/stellar/stellarsvc/service.go

```diff
@@ -155,6 +156,23 @@ func postFromCurrentUser(ctx context.Context, g *libkb.GlobalContext, acctID ste
 	return post
 }
 
+type seqnoProv struct {
```

Comment:
This could be up one level in `stellar.SeqnoProv` since it's reusable, and so `stellarsvc` is slim